extensions/wikibase/src/org/openrefine/wikibase/qa/EditInspector.java

```diff
@@ -215,6 +217,11 @@ public void inspect(List<EntityEdit> editBatch, WikibaseSchema schema) throws Ex
     }
 
     private void resolveWarningPropertyLabels() throws ExecutionException {
+        List<String> warningTypeswithSuggestedData = List.of("new-item-requires-property-to-have-certain-values",
+                "new-item-requires-certain-other-statement",
+                "existing-item-requires-property-to-have-certain-values",
+                "existing-item-requires-certain-other-statement");
```

Comment:
I would expect that we don't need to list the constraint types here: as discussed earlier, the later code could directly inspect the entities that are returned as part of the warnings, and add the labels if they are missing.

Reply:
Instead of adding an item id value inside a property id value, I think the scrutinizer should add two separate parameters to the warning it creates: one for the property and one for the item.

Reply:
@wetneb Can you also confirm if the logic for identifying the expected value is correct?

Reply:
no, please just read the expected value from the constraints definition and pass it as a separate warning parameter (independently of the property id). This probably requires changing the warning id as well.

Reply:
@wetneb  thanks for the clarification, i had ended up re-inventing the constraint flow.. 
One query, the value definition in ItemRequiresConstraint is a List, what needs to be done if there are multiple values.